src/EditorFeatures/CSharpTest/RemoveUnusedMembers/RemoveUnusedMembersTests.cs

```diff
@@ -1086,33 +1092,41 @@ public async Task PropertyIsOnlyWritten()
             var source =
 @"class MyClass
 {
-    private int [|P|] { get; set; }
+    private int {|IDE0052:P|} { get; set; }
     public void M()
     {
         P = 0;
     }
 }";
-            var testParameters = new TestParameters(retainNonFixableDiagnostics: true);
-            using var workspace = CreateWorkspaceFromOptions(source, testParameters);
-            var diagnostics = await GetDiagnosticsAsync(workspace, testParameters).ConfigureAwait(false);
-            diagnostics.Verify(Diagnostic("IDE0052", "P").WithLocation(3, 17));
-            var expectedMessage = string.Format(FeaturesResources.Private_property_0_can_be_converted_to_a_method_as_its_get_accessor_is_never_invoked, "MyClass.P");
-            Assert.Equal(expectedMessage, diagnostics.Single().GetMessage());
+
+            await new VerifyCS.Test
+            {
+                TestCode = source,
+                ExpectedDiagnostics =
+                {
+                },
+                FixedCode = source,
+            }.RunAsync();
+            //var testParameters = new TestParameters(retainNonFixableDiagnostics: true);
```

Comment:
Is message validation an unsupported scenario? Any case, this test needs to be fixed to remove the commented out code, and if required skipped with a tracking issue to unskip.

Reply:
> Is message validation an unsupported scenario?

Messages are always validated for diagnostics in `ExpectedDiagnostics`, but ignored for diagnostics in markup.

Reply:
➡️ Fixed in cfbdcec06aaefbd2a6b8e94dda30bddffe82e431